docs/outlook/delegate-access.md

```diff
@@ -152,15 +169,11 @@ if (item.getSharedPropertiesAsync) {
 
 ## Limitations
 
-Depending on your add-in's scenarios, there are a couple of limitations for you to consider when handling delegate situations.
-
-### REST and EWS
-
-Your add-in can use REST but not EWS, and the add-in's permission must be set to `ReadWriteMailbox` to enable REST access to the owner's mailbox.
+Depending on your add-in's scenarios, there are a few limitations for you to consider when handling delegate situations.
 
 ### Message Compose mode
 
-In Message Compose mode, [getSharedPropertiesAsync](/javascript/api/outlook/office.messagecompose#getSharedPropertiesAsync_options__callback_) is not supported in Outlook on the web or Windows unless the following conditions are met.
+In Message Compose mode, [getSharedPropertiesAsync](/javascript/api/outlook/office.messagecompose#getSharedPropertiesAsync_options__callback_) is not supported in Outlook on the web or on Windows unless the following conditions are met.
```

Comment:
Compose mode is special in the way that if the assistant created the response item(new mail,reply,fwd), no matter which mailbox they created this in, the compose item is from assistant's mailbox and hence they are the owner. This item is not a sharedItem. 
If the manager created a new compose form and then shared the folder with the assistant that contains this draft item, when the assistant navigates to that sharedFolder, only now is that draft a sharedItem and getSharedProperties will work on it. This also means that only addins that have the manifest tag will run on such an item. In the former case all addins will run on the item since the assistant was the owner of the item, we don't filter out addins based on manifest tag.

Reply:
[anoberoi] to follow up on the following behavior: However, if a user composes a mail, saves as draft, and moves the draft into the shared mailbox’s draft, then the item becomes a shared item. In this case, if a different user on the shared mailbox accesses the draft and continues to edit/send, it would be a proper “on-send for shared mailboxes” scenario, and in this scenario the on-send events still work and are supported in the shared mailbox (Although this should be noted as working in preview and not production).

Reply:
For this entire "Limitations" section, let's determine delegate access vs shared mailbox behavior then we can document accordingly